doc/in-progress/time.md

```diff
@@ -0,0 +1,59 @@
+# Accessing Time in Move
+
+## `sui::clock::Clock`
+
+Transactions that need access to an accurate clock must pass a read-only reference of `sui::clock::Clock` in as an entry function parameter.  The only available instance is found at address `0x6`.
+
+A unix timestamp in milliseconds can be extracted from an instance of `Clock` using
+
+```
+module sui::clock {
+    public fun timestamp_ms(clock: &Clock): u64;
+}
+```
+
+**The resulting value is expected to change every 2 to 3 seconds**, at the rate that checkpoints are committed by the network.
+
+Successive calls to `sui::clock::timestamp_ms` in the same transaction will always produce the same result (transactions are considered to take effect instantly), and may also overlap with other transactions that touch the same shared objects, so it is not safe to assume that time progresses between transactions that are sequenced relative to each other.
+
+Any transaction that requires access to a `Clock` must go through **consensus**, because the only available instance is a **shared object**, so this technique is not suitable for transactions that must use the single-owner fast-path (see "Epoch timestamps" for a single-owner-compatible source of timestamps).
```

Comment:
```suggestion
Any transaction that requires access to a `Clock` must go through [consensus](/learn/architecture/consensus) because the only available instance is a [shared object](/learn/objects#shared). As a result, this technique is not suitable for transactions that must use the single-owner fast-path (see [Epoch timestamps](#epoch-timestamps) for a single-owner-compatible source of timestamps).
```

Reply:
I will update these links to github paths once this page has a home.